package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Enact JavaScript Framework Docs",
   "version": "4.0.1",
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=14"
```

Comment:
Do we fix all issues over node 17?

Reply:
Yes.
Currently, travis check in the docs develop branch passes without problems on nodes 18 and 19.
https://github.com/enactjs/docs/pull/207/checks